server/test/handlers/inputSchool.test.js

```diff
@@ -0,0 +1,110 @@
+const schoolHandler = require('../../models/handlers/school');
+
+describe('testing invalid schoolId parameters', () => {
+    beforeEach(() => {
+        jest.resetModules(); // Clears any cache between tests.
+    });
+
+    test('rejection of no parameters', async () => {
+        // Arrange
+        // No test variables need to be initialized.
+
+        // Act
+        const isSuccessfullyInserted = await schoolHandler.getWorkerIdsForSchool();
```

Comment:
HI Simran,
Please make sure to run the unit tests in the server folder to make sure they pass. Based on the current code, the getWorkerIdsForSchool returns an array of workerIds as opposed to a boolean value.